docs/decisions/0005-usage-of-reactstrap.rst

```diff
@@ -0,0 +1,55 @@
+5. Usage of Reactstrap
+----------------------
+
+Status
+------
+
+Proposed
+
+Context
+-------
+
+Paragon components exist for a variety of use cases, are React-based, and are currently styled using Bootstrap-compatible CSS class names. All known usages of Paragon are backed by Bootstrap. Reactstrap is a React-based component library that provides implementations of Bootstrap's components.
+
+Paragon's strength is found in its ability to be modified to meet our accessibility and functionality requirements above and beyond what any third-party component library might contain.  In addition, Paragon is the home for  edX-specific reusable components that would never be found in a generic component library.
```

Comment:
I'm not suggesting that we remove the `edx-specific` qualifier here, but I'm wondering what's an example of something in Paragon today that's edx-specific?  I do imagine that we will be creating reusable _organisms_ that are edx-specific - but we had been thinking of not including those in Paragon itself and keeping Paragon for low-level _atoms_ and _molecules_ only.  I realize that such a decision (however we decide) will need a separate ADR.  But in the meantime, do we already have edx-specific components in Paragon today?

Reply:
It's a fair point.  When I wrote it I was assuming there were some paradigms/features that found their way into Paragon that were fulfilling particular design requirements on our site.  Looking around paragon in more detail, I can't really find anything that fits that bill at the moment.

I like the distinction as you frame it.  If we simply say that any "edx-specific components" are really organisms _because_ they're specific to our business and our particular domain of data, then that excludes them from Paragon by definition.

As an example, a reusable "certificate card" that displays a cert that a user has earned would then be classified as an organism and (practically speaking) would then need its own repository or would be pushed into some non-paragon repo.  

I think that holds together.  Yes?  (in which case I _will_ remove that language above)